databricks_cli/dbfs/api.py

```diff
@@ -230,13 +146,11 @@ def get_num_files_deleted(partial_delete_error):
                     message))
         return int(m.group(1))
 
-    @Retry429
     def delete(self, dbfs_path, recursive, headers=None):
         num_files_deleted = 0
         while True:
             try:
-                self.client.delete(dbfs_path.absolute_path, 
-                                   recursive=recursive, headers=headers)
+                self.client.delete(dbfs_path.absolute_path, recursive=recursive, headers=headers)
             except HTTPError as e:
                 if e.response.status_code == 503:
```

Comment:
https://github.com/urllib3/urllib3/blob/3308d655a563e0b72e3856c4a4cb06ce2f4f0e8d/src/urllib3/util/retry.py#L217 `HTTP 503` would be retried globally, please check conditions

Reply:
Why do you say there will be global retries on 503? I am overriding the default status codes with `status_forcelist=[429]` in `databricks_cli/sdk/api_client.py` to ensure retries are made only for `429`.
```
#: Default status codes to be used for ``status_forcelist``
    RETRY_AFTER_STATUS_CODES = frozenset([413, 429, 503])
```

Reply:
then good